src/base/utils/misc.cpp

```diff
@@ -619,18 +621,26 @@ void Utils::Misc::openFolderSelect(const QString &absolutePath)
         ::CoUninitialize();
 #elif defined(Q_OS_UNIX) && !defined(Q_OS_MAC)
     QProcess proc;
-    proc.start("xdg-mime", QStringList() << "query" << "default" << "inode/directory");
+    proc.start("xdg-mime", {"query", "default", "inode/directory"});
     proc.waitForFinished();
     QString output = proc.readLine().simplified();
     if ((output == "dolphin.desktop") || (output == "org.kde.dolphin.desktop"))
-        proc.startDetached("dolphin", QStringList() << "--select" << Utils::Fs::toNativePath(path));
+        proc.startDetached("dolphin", {"--select", Utils::Fs::toNativePath(path)});
     else if ((output == "nautilus.desktop") || (output == "org.gnome.Nautilus.desktop")
-             || (output == "nautilus-folder-handler.desktop"))
-        proc.startDetached("nautilus", QStringList() << "--no-desktop" << Utils::Fs::toNativePath(path));
+                 || (output == "nautilus-folder-handler.desktop")) {
```

Comment:
Coding Guidelines:
>Notice that for a series of `if - else` branches, if one branch needs braces then all branches must add braces.

So add braces to other branches too.

Reply:
@glassez commit "fix branches for a series of "if - else"", add other branches for a series of if - else. if everything is ok, combine commits